File: ErrorLog.cs
using System;
using Microsoft.VisualBasic;

namespace BossAdmin
{
    public partial class ErrorLog
    {
        public ErrorLog()
        {
            InitializeComponent();
        }

        private void btnCopy_Click(object sender, EventArgs e)
        {
            try
            {
                My.MyProject.Computer.Clipboard.SetText(uteLog.Text);
                Interaction.MsgBox("Process Log) is saved to your clipboard.  You can paste it into a document or directly into an email", MsgBoxStyle.Information, "Copy to Clipboard");
            }

            catch (Exception ex)
            {

            }
        }

        private void btnPrint_Click(object sender, EventArgs e)
        {
            {
                var withBlock = UltraPrintDocument1;
                withBlock.Header.TextCenter="Cost Analysis";
                withBlock.Footer.TextCenter=uteLog.Text;
                withBlock.Print();
            }

        }
    }
}

File: ErrorLog.vb
Public Class ErrorLog

    Private Sub btnCopy_Click(sender As Object, e As EventArgs) Handles btnCopy.Click
        Try
            My.Computer.Clipboard.SetText(uteLog.Text)
            MsgBox("Process Log) is saved to your clipboard.  You can paste it into a document or directly into an email", MsgBoxStyle.Information, "Copy to Clipboard")

        Catch ex As Exception

        End Try
    End Sub

    Private Sub btnPrint_Click(sender As Object, e As EventArgs) Handles btnPrint.Click
        With UltraPrintDocument1
            .Header.TextCenter = "Cost Analysis"
            .Footer.TextCenter = uteLog.Text
            .Print()
        End With

    End Sub
End Class